src/cf-base.less

```diff
@@ -536,26 +506,18 @@ table {
 
 th,
 td {
-    padding: unit(12px / @base-font-size-px, em) unit(15px / @base-font-size-px, em);
-    background: @td-bg;
+    padding: 10px;
 
     thead & {
         color: @thead-text;
         background: @thead-bg;
-    }
-
-    tbody > tr:nth-child(odd) > & {
-        background: @td-bg-alt;
-    }
-
-    .compact-table & {
-        padding: unit(7px / @base-font-size-px, em) unit(10px / @base-font-size-px, em);
+        .h5();
     }
 }
 
-th {
-    .webfont-demi();
-    text-align: left;
+thead,
+tbody tr {
```

Comment:
This is an intentional omission. I confirmed with @niqjohnson - if, for some odd reason, there were multiple rows in the THEAD, there should only be one border under the entire header.